app/src/main/java/com/abhijitvalluri/android/fitnotifications/AppSettingsActivity.java

```diff
@@ -186,16 +190,18 @@ public void onBackPressed() {
     public void onActivityResult2(int requestCode, Intent data) {
         int hour = data.getIntExtra(TimePickerFragment.EXTRA_HOUR, 0);
         int minute = data.getIntExtra(TimePickerFragment.EXTRA_MINUTE, 0);
-        if (requestCode == START_TIME_REQUEST) {
+
+        java.text.DateFormat timeFormat = DateFormat.getTimeFormat(this);
+        String formattedTime = timeFormat.format(Func.convertHourMinute2Date(hour, minute));
+
+        if (requestCode == START_TIME_REQUEST) {`
```

Comment:
It used to add *.iml to .gitignore indeed. But apparently just a week ago (!) IntelliJ have updated their [recommendations](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839):
> Here is what you need to share:
> ...
> - All the .iml module files that can be located in different module directories

Would you mind removing `*.iml` from `.gitignore` and adding *.iml files to Git?

Also please add `/.idea/gradle.xml` to `.gitignore` and remove it from version control as it stores user-specific path in "gradleHome" property.